martor/templates/martor/editor.html

```diff
@@ -17,6 +17,22 @@
     <div class="ui bottom attached tab segment martor-preview" data-tab="preview-tab-{{ field_name }}">
       <p>{% trans "Nothing to preview" %}</p>
     </div>
+    {% if editor_msg or button_text %}
+      <hr>
+    {% endif %}
+    {% if button_text %}
+      <div class="ui tab show-on-editor active">
+        <input style="float: right" type="submit" value="{{ button_text }}" class="button disabled" disabled>
+      </div>
+      <div class="ui tab show-on-preview">
+        <input style="float: right" type="submit" value="{{ button_text }}" class="button">
+      </div>
+    {% endif %}
+    {% if editor_msg %}
+      <div class="ui tab show-on-editor active">
```

Comment:
Would it be difficult to make this message a tooltip when hovering the button on desktop/browsers with tooltip hover support? Would need a fallback for mobile. Ignore if this is too annoying to implement.

Reply:
tooltips are annoying to implement. it is possible to do `<input ... disabled title="foo">` but this is too hidden to be useful.